composites/Plugin/SnippetEditor/components/ModeSwitcher.js

```diff
@@ -54,8 +53,6 @@ const DesktopButton = SwitcherButton.extend`
 const Switcher = styled.div`
 	display: inline-block;
 	margin-top: 10px;
-	margin-right: ${ getRtlStyle( "0px", "20px" ) };
-	margin-left: ${ getRtlStyle( "20px", "4px" ) };
 	border: 1px solid #dbdbdb;
```

Comment:
The 4px margin was needed for RTL, though a better fix would be reversing the Edit Snippet button margin.